amplify/backend/function/expressServer/src/controllers/portfolioController.js

```diff
@@ -1,30 +1,23 @@
 const User = require('../models/user.model')
-const Portfolio = require('../models/portfolio.model')
+const Portfolio = require('../models/portfolio.model');
+const PortfolioService = require('../services/portfolioServices');
 
 
 
-// @desc create new portfolio
+// @desc create new hanging portfolio
 // route post portfolio/createPortfolio
 // @access private
 
-const createPortfolio = async (req, res, next) => {
+const createHangingPortfolio = async (req, res, next) => {
   try {
-      dateCreated = new Date()
-      if (!req.body.remainder){
-        remainder = 10000
-      }
-      else {
-        remainder = req.body.remainder
-      }
-      const newPortfolio = new Portfolio({
+    // create object to be used in createPortfolio service
+      const portfolioData = {
```

Comment:
We need to have a check here for required fields. 

When I send no data in body the app crashes. 
![image](https://user-images.githubusercontent.com/61060096/200601780-37c0e066-0b2b-41ee-89cd-6dce6d97e735.png)
![image](https://user-images.githubusercontent.com/61060096/200601722-75596e59-b7ca-4847-843b-d7bcffc27a38.png)

When I send only a portflioName I can create it but it has no balance. 
![image](https://user-images.githubusercontent.com/61060096/200602249-66dde03f-b101-486a-917e-8d266e2ecaad.png)
![image](https://user-images.githubusercontent.com/61060096/200602426-c28622f1-4ff5-4e33-a5d1-753d557afa9a.png)

When I send a balance only it returns a 200 status but no portfolio is created? 
![image](https://user-images.githubusercontent.com/61060096/200603444-7064bea5-ad79-4237-8498-fa044033b2a7.png)
![image](https://user-images.githubusercontent.com/61060096/200603102-264ac09d-3721-4647-935f-a4fa757957b6.png)

If I send wrong data type app crashes. 
![image](https://user-images.githubusercontent.com/61060096/200604054-4ad66e69-2ba5-4371-8733-9f72aaf565eb.png)
![image](https://user-images.githubusercontent.com/61060096/200604133-58649f36-dd7d-48ac-95d9-fb86cd31a3b7.png)